core/src/replay_stage.rs

```diff
@@ -963,10 +950,8 @@ impl ReplayStage {
         bank: &Bank,
         blocktree: &Blocktree,
         bank_progress: &mut ForkProgress,
-    ) -> Result<(Vec<Entry>, usize, bool)> {
-        blocktree
-            .get_slot_entries_with_shred_info(bank.slot(), bank_progress.num_shreds as u64)
-            .map_err(|err| err.into())
+    ) -> result::Result<(Vec<Entry>, usize, bool), BlocktreeError> {
```

Comment:
We can delete this `load_blocktree_entries_with_shred_info` function and just use the definition inline